jdaviz/app.py

```diff
@@ -591,6 +591,14 @@ def add_data(self, data, data_label):
             f"Data '{data_label}' successfully added.", sender=self)
         self.hub.broadcast(snackbar_message)
 
+    def add_subset(self, subset, subset_label):
+        """Add subset to Glue ``DataCollection``."""
+
+        # TODO: Need translation magic from Astropy region to Glue subset
```

Comment:
This probably belongs in glue-astronomy - we currently implement the reverse translation there:

https://glue-astronomy.readthedocs.io/en/latest/translators.html#selection-information

Specifically, we should probably extend the translation machinery in glue:

https://docs.glueviz.org/en/latest/customizing_guide/customization.html#custom-translation-of-subset-definitions-to-native-data-objects

to include a ``to_subset`` for the subset translation.

Reply:
Thanks! I'll keep this in mind if this make it into a Sprint.

Reply:
Roundtripping not needed, so we don't have to discuss this any further.